cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-2.launch

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<launch>
```

Comment:
this launch file includes files that do not exist in ipa320/indigo_dev anymore!
have a look at default_controllers_cob3-6.launch